nvflare/private/fed/utils/numproto.py

```diff
@@ -73,6 +73,4 @@ def proto_to_bytes(nda_proto: NDArray) -> bytes:
     Returns:
         Returns bytes.
     """
-    nda_bytes = BytesIO(nda_proto.ndarray)
-
-    return nda_bytes.read()
+    return nda_proto.ndarray
```

Comment:
Is this right? This func is supposed to return bytes, but you are now returning NDArray.